kubeflow/trainer/rhai/utils_test.py

```
@@ -133,5 +145,371 @@ def test_parse_output_dir_uri_with_pvc(test_case):
+    """Test setup_rhai_trainer_storage handles trainer without output_dir."""
+    print("Executing test: setup_rhai_trainer_storage_no_output_dir")
+
+    mock_core_api = MagicMock()
+
+    # Create a mock trainer without output_dir
+    mock_trainer = MagicMock(spec=[])  # No output_dir attribute
+
+    mock_trainer_cr = MagicMock()
+    mock_trainer_cr.env = []
+
+    resolved_dir, result_cr, result_overrides = setup_rhai_trainer_storage(
```

Ruff (0.14.13)

Unpacked variable `result_cr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)